src/fadec/src/EngineControl.h

```diff
@@ -202,8 +202,18 @@ class EngineControl {
       }
     }
 
-    // Present State Starting
+    // Present State Starting. Fuel used reset to zero
     if (EngineState == 2) {
```

Comment:
Add a TODO comment here for the future once we have the DMC system implemented.

i.e. Fuel used is reset when Engine started on ground _AND_ 3 DMCs (display management controllers) are reset (powered off for >= 5 s).

Reply:
I do not think the todo should be added here. The DMC will be a separate module and will need separate code to initiate a fuel used reset. The engine restart and the DMC reset are not AND, they are OR.

This process was tested **today on a real A320** by a A320 pilot. The fuel used was reset at the time the engine was restarted on ground. The fuel used is a simple sum of the current fuel used by both engines. Total fuel used reduced when the engine was restarted and that engine fuel used reset to zero.

I know there is concern about where the code should go. This location  works as expected in the real world.

If the QA follows the expected behavior in the testing instructions. Then this fix will simulate the observed outcome seen today on an A320.

Reply:
I do not think that the TODO should be added here. The engine restart and DMC reset are OR events not AND events.
The DMC will have its own code and any code or TODO needed to reset fuel used should be in that module.

The engine reset was tested **today on a real A320** the engine was shut down and restarted. The fuel used was reset to zero. The total fuel used reduced to reflect the change of the fuel used reset. 

I know there are concerns about the location of this code, but it works as seen today in real life.

If the QA follows the testing instructions then this fix simulates what was seen today on an A320.